qa-core/src/config/internal-api/TestConfiguration/userManagement.ts

```diff
@@ -0,0 +1,117 @@
+import { Response } from "node-fetch"
+import { Role, User, UserDeletedEvent } from "@budibase/types"
+import InternalAPIClient from "./InternalAPIClient"
+import { responseMessage } from "../fixtures/types/responseMessage"
+
+export default class UserManagementApi {
+    api: InternalAPIClient
+
+    constructor(apiClient: InternalAPIClient) {
+        this.api = apiClient
+    }
+
+    async searchUsers(): Promise<[Response, User[]]> {
```

Comment:
We could update these method names to be more inline with the rest of the API clients:
- `search`
- `fetchAll`
- `deleteMultiple`

etc, because they will be used in the tests like so:

`users.search` instead of `users.searchUsers`, which is a bit redundant.